markets/perps-market/test/integration/Orders/OffchainAsyncOrder.settle.test.ts

```diff
@@ -1,102 +1,77 @@
 import { ethers } from 'ethers';
-import { bn, bootstrapMarkets } from '../bootstrap';
-import { fastForwardTo } from '@synthetixio/core-utils/utils/hardhat/rpc';
+import { DEFAULT_SETTLEMENT_STRATEGY, bn, bootstrapMarkets } from '../bootstrap';
+import { fastForwardTo, getTime } from '@synthetixio/core-utils/utils/hardhat/rpc';
 import assertBn from '@synthetixio/core-utils/utils/assertions/assert-bignumber';
 // import assertEvent from '@synthetixio/core-utils/utils/assertions/assert-event';
 import assertRevert from '@synthetixio/core-utils/utils/assertions/assert-revert';
 import { commitOrder } from '../helpers';
 
-const ASYNC_OFFCHAIN_ORDER_TYPE = 1;
-const ASYNC_OFFCHAIN_URL = 'https://fakeapi.pyth.synthetix.io/';
-
 describe('Settle Offchain Async Order test', () => {
-  const { systems, marketOwner, perpsMarkets, provider, trader1, keeper } = bootstrapMarkets({
+  const { systems, perpsMarkets, provider, trader1, keeper } = bootstrapMarkets({
     synthMarkets: [],
-    perpsMarkets: [{ name: 'Ether', token: 'snxETH', price: bn(1000) }],
+    perpsMarkets: [
+      {
+        name: 'Ether',
+        token: 'snxETH',
+        price: bn(1000),
+        fundingParams: { skewScale: bn(100_000), maxFundingVelocity: bn(0) },
+      },
+    ],
     traderAccountIds: [2, 3],
   });
-
-  const settlementDelay = 5;
-  const settlementWindowDuration = 120;
-  const settlementReward = bn(5);
-  const priceDeviationTolerance = bn(0.01);
-
-  const feedId = ethers.utils.formatBytes32String('ETH/USD');
-
-  let priceVerificationContract: string;
   let marketId: ethers.BigNumber;
 
   before('identify actors', async () => {
     marketId = perpsMarkets()[0].marketId();
-    priceVerificationContract = systems().MockPyth.address;
-  });
-
-  before('create settlement strategy', async () => {
-    await systems().PerpsMarket.connect(marketOwner()).addSettlementStrategy(marketId, {
-      strategyType: ASYNC_OFFCHAIN_ORDER_TYPE, // OFFCHAIN
-      settlementDelay,
-      settlementWindowDuration,
-      priceVerificationContract,
-      feedId,
-      url: ASYNC_OFFCHAIN_URL,
-      disabled: false,
-      settlementReward,
-      priceDeviationTolerance,
-    });
-  });
-
-  before('set skew scale', async () => {
-    await systems()
-      .PerpsMarket.connect(marketOwner())
-      .setFundingParameters(marketId, bn(100_000), bn(0));
   });
 
   before('add collateral', async () => {
     await systems().PerpsMarket.connect(trader1()).modifyCollateral(2, 0, bn(10_000));
   });
 
-  const { startTime } = commitOrder(
-    {
-      trader: trader1,
-      marketId: () => marketId,
-      accountId: () => 2,
-      sizeDelta: () => bn(1),
-      settlementStrategyId: () => 0,
-      settlementDelay: () => settlementDelay,
-      settlementWindowDuration: () => settlementWindowDuration,
-      acceptablePrice: () => bn(1_000),
-      trackingCode: ethers.constants.HashZero,
-    },
-    { systems, provider }
-  );
```

Comment:
commitOrder now runs inside a before block.  with this change, we don't have to make all the variables functions